src/java/lesson3/TaskThree.java

```diff
@@ -0,0 +1,15 @@
+package lesson3;
+
+public class TaskThree {
+    public static void main(String[] args) {
+        int month = 15;
+        String s1 = switch (month) {
+                case 1, 2, 12 -> "Winter";
```

Comment:
по можливості давай описові назви 
```suggestion
        String result = switch (month) {
                case 1, 2, 12 -> "Winter";
```